### DISCUSSION TITLE
Maandag 16-12-24

### DISCUSSION THREAD

**Discussion by Nazneen05x** [rutgerkock/buurtcampus-oost#19] [Stand-ups]
- Sprint doel behaald?
- Hoe gaat het met je en waar ben je nú mee bezig?
- Wat wil je deze week bereiken (aan werken)?
- Waar loop je tegen aan?

**Comment by Nazneen05x** [comment]
### Sprint doel behaald?

Begin van deze sprint had ik aangegeven dat ik het menu en over pagina af wil hebben. 

### Menu:

Het menu is helemaal gebouwd. Mobiele versie, desktop versie en dropdown menu is gebouwd. 

Ik heb nog een bug probleem en dat is responsief op mijn mobiele telefoon. Het is altijd een hele klus om het goed responsief te krijgen voor mijn mobiel (Huawei). De componenten Harry en AccessibilityMenu, veroorzaakt wat problemen in mijn mobiele menu. Komt omdat deze twee componenten niet goed responsief zijn op mijn mobiel. Ik moet nog testen of andere mobiele telefoons deze probleem hebben.


<img width="150" alt="Scherm­afbeelding 2024-09-16 om 14 51 22" src="https://github.com/user-attachments/assets/e6ccf8e0-61e6-4143-ae79-9076a0e9d390">

<img width="150" alt="Scherm­afbeelding 2024-09-16 om 14 51 22" src="https://github.com/user-attachments/assets/3bb32a0d-4d06-4d2c-8505-e13ba81739e1">

-----
<img width="150" alt="Scherm­afbeelding 2024-09-16 om 14 51 22" src="https://github.com/user-attachments/assets/2168d8e9-b826-4dc4-ac85-8c0bea07e19a">

 **Het hoort er zo uit te zien, dit is wanneer ik het heb uitgezoomd op mijn telefoon.**


### TO DO:
Testen testen testen.
bug moet opgelost worden.

### Over pagina:

De Over pagina is helemaal gebouwd! Ik moet nu nog alleen testen op verschillende devices. 

### Hoe gaat het met je en waar ben je nú mee bezig?

🫠

Vorige week was ik vooral bezig met testen en (coderen menu & over pagina).

### Wat wil je deze week bereiken (aan werken)?

Over pagina:

- [ ]  Testen afronden - Als responsief niet in orde is, dan verbeteren.
- [ ]  Pull request.

Menu:

- [ ] Testen testen.
- [ ] Ik weet niet of deze week de bug nog opgelost kan worden. Na de vakantie. 

Partners & Groene bieb:

- [ ] Deze week wil ik een begin maken aan deze twee pagina’s.
- [ ] Testplan!!



### Waar loop je tegen aan?

Te veel om op te noemen . Het is tijd voor vakantie!✈️

**Comment by Annevd** [comment]
### Sprint doel behaald?
Ik gaf aan dat ik alle must haves van de contact pagina af wilde hebben, en een begin wilde maken aan de zaden pagina. Alle must haves van de contact pagina zijn af, dus dat is gelukt! (op het regelen van het emailadres na, maar daar ben ik mee bezig). Ik heb helaas nog geen tijd gevonden om te beginnen aan de zaden pagina, dus daar zal mijn focus op liggen in sprint 18.
### Hoe gaat het met je en waar ben je nú mee bezig?
Het gaat wel goed, ben druk bezig met creative coding, mentoren en portflow beetje bij beetje invullen.
### Wat wil je deze week bereiken (aan werken)?
Ik ga weer bezig aan een nieuwe creative coding opdracht dus daar gaat de meeste tijd in zitten. Maaarrrrrrrrrr, woensdagochtend gaan we testen met echte buurtbewonders! 
### Waar loop je tegen aan?
Tijd vinden voor het projectplan. Maar volgende sprint hebben we alle tijd en ruimte om te knallen!
